lib/matplotlib/axes/_base.py

```diff
@@ -2172,7 +2172,15 @@ def update_datalim_bounds(self, bounds):
         self.dataLim.set(mtransforms.Bbox.union([self.dataLim, bounds]))
 
     def _process_unit_info(self, xdata=None, ydata=None, kwargs=None):
-        """Look for unit *kwargs* and update the axis instances as necessary"""
+        """
+        Look for unit *kwargs* and update the axis instances as necessary
+
+
+        .. warning ::
+
+           This method may mutate the dictionary passed in an kwargs and
+           the Axes instances attached to this Axes.
```

Comment:
"passed in an kwargs" -> "passed in as kwargs"
"the Axes instances" -> "the Axis instances"